src/input_json_format/cfdapi_get_schnorr_pubkey_from_privkey copy.json

```
@@ -0,0 +1,21 @@
+{
+  "namespace": ["cfd", "js", "api", "json"],
+  "comment": "Get a Schnorr pubkey from a privkey.",
+  "request": {
+    ":class": "GetSchnorrPubkeyFromPrivkeyRequest",
+    ":class:comment": "Request to get a Schnorr pubkey from privkey.",
+    "privkey": "",
+    "privkey:require": "require",
+    "privkey:comment": "privkey (wif or hex)",
+    "isCompressed": true,
+    "isCompressed:require": "optional",
+    "isCompressed:comment": "compressed pubkey flag"
```

unused `isCompressed`.
```suggestion
    "privkey:comment": "privkey (wif or hex)"
```

---

Thanks fixed (also fixed the name of the file).

---

Thanks fixed (also fixed the file name)